src/test/ui/lint/pineapple-on-pizza.stderr

```diff
@@ -0,0 +1,13 @@
+error: pineapple doesn't go on pizza
+  --> $DIR/pineapple-on-pizza.rs:5:18
+   |
+LL |     let _: Pizza<Pineapple>;
+   |            ----- ^^^^^^^^^
+   |            |
+   |            this is the pizza you ruined
+   |
+   = note: `#[forbid(pineapple_on_pizza)]` on by default
```

Comment:
Given the atrocious crime against taste that has been committed here, I wonder if we should delete the file from the user's hard-drive. Although, I'm not sure how to test that... 🤔

Reply:
We could get close by obtaining the whole span of the file and adding a removal suggestion to remove the complete span. Since we have rustfix tests  this should be easily testable

Reply:
Ah that's clever! @pietroalbini Can you adjust the PR to do that?

Reply:
Can you get the span of the whole crate?  It's the only way to be sure.